debezium-core/src/main/java/io/debezium/relational/TableIdTransformerQuoted.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.relational;
+
+/**
+ * Generic implementation of a TrableIdTransformer that accepts either
+ * a quote character to use for the start and end of every identifier
+ * part or two separate identifiers to use for start and end of each
+ * part.
+ *
+ */
+public class TableIdTransformerQuoted implements TableIdTransformer {
+
```

Comment:
Okay, I guess the name makes a bit more sense to me now. Perhaps the instances I mentioned above could go in `TableIdTransformer`??